builder.py

```diff
@@ -225,14 +234,11 @@
 			for b in base:				
 				if b['name'] == 'PAYMENT' and any([[True for x in [deploy['name'] for deploy in input_template[0]['daemons']] if x==xx] for xx in [echain['name'] for echain in evm_chains]]):
 					print(f"[bold magenta]{'-'*50}[/bold magenta]")
-					if cache["payment_tier1"]:
-						b['payment_tier1'] = cache["payment_tier1"]
-					if cache["payment_tier2"]:
-						b['payment_tier2'] = cache["payment_tier2"]
-					if cache["discount_ablock"]:
-						b['discount_ablock'] = cache["discount_ablock"]
-					if cache["discount_aablock"]:
-						b['discount_aablock'] = cache["discount_aablock"]
+					b['payment_tier1'] = cache["payment_tier1"] if cache["payment_tier1"] != None else 35
+					b['payment_tier2'] = cache["payment_tier2"] if cache["payment_tier2"] != None else 200
+					b['discount_ablock'] = cache["discount_ablock"] if cache["discount_ablock"] != None else 20
+					b['discount_aablock'] = cache["discount_aablock"] if cache["discount_aablock"] != None else 0
+					b['discount_sysblock'] = cache["discount_sysblock"] if cache["discount_sysblock"] != None else 10
```

Comment:
These lines (237-241) should actually be like this:
```python
b['payment_tier1'] = cache["payment_tier1"] if cache["payment_tier1"] != None else b['payment_tier1']
b['payment_tier2'] = cache["payment_tier2"] if cache["payment_tier2"] != None else b['payment_tier2']
b['discount_ablock'] = cache["discount_ablock"] if cache["discount_ablock"] != None else b['discount_ablock']
b['discount_aablock'] = cache["discount_aablock"] if cache["discount_aablock"] != None else b['discount_aablock']
b['discount_sysblock'] = cache["discount_sysblock"] if cache["discount_sysblock"] != None else b['discount_sysblock']
```
The reason is because there should be no default values hard coded in the python. All payment-related default values are defined in the PAYMENT section of `autobuild/sources.yaml`. After making these changes, the default values defined in `autobuild/sources.yaml` will be used when no values have yet been set for these payment items in `.cache`.

Of course this little section could be optimized even further like this:
```python
if cache["payment_tier1"] != None: b['payment_tier1'] = cache["payment_tier1"] 
if cache["payment_tier2"] != None: b['payment_tier2'] = cache["payment_tier2"] 
if cache["discount_ablock"] != None: b['discount_ablock'] = cache["discount_ablock"] 
if cache["discount_aablock"] != None: b['discount_aablock'] = cache["discount_aablock"] 
if cache["discount_sysblock"] != None: b['discount_sysblock'] = cache["discount_sysblock"] 
```

Reply:
Done. Also the branchpath regexp.
LGTM!